### PR TITLE
feat: Allows the selection of key's params using rdvp.

### PR DESCRIPTION
**Pull request by Jorropo**
Also changed the default to `Ed25519` not `RSA` (should be cheaper to compute for mobile).
I think it's worth to change the main rdvp server to `Ed25519` or `ECDSA` now (while not many people are testing berty) (because that change the peer ID).

### CODE REVIEW

File: go/cmd/rdvp/main.go [hunk 1]
```diff
@@ -89,7 +94,8 @@ func main() {
 					return errcode.TODO.Wrap(err)
 				}
 			} else {
-				priv, _, err = libp2p_ci.GenerateKeyPairWithReader(libp2p_ci.RSA, 2048, crand.Reader) // nolint:staticcheck
+				// Don't use key params here, this is a dev tool, a real instalations should use a static key.
```
Comment (moul) [inline go/cmd/rdvp/main.go:97]: s/instalations/installation

**Review by moul — APPROVED** [review]
And yes, @gfanton, can you update the current dev server's key?

File: go/cmd/rdvp/main.go (hunk 1 above)
Comment (Jorropo) [reply go/cmd/rdvp/main.go]: @moul done :)

**Review by moul — APPROVED** [review]

File: go/cmd/rdvp/main.go [hunk 2]
```diff
@@ -89,7 +94,8 @@ func main() {
 					return errcode.TODO.Wrap(err)
 				}
 			} else {
-				priv, _, err = libp2p_ci.GenerateKeyPairWithReader(libp2p_ci.RSA, 2048, crand.Reader) // nolint:staticcheck
+				// Don't use key params here, this is a dev tool, a real installation should use a static key.
+				priv, _, err = libp2p_ci.GenerateKeyPairWithReader(libp2p_ci.Ed25519, 2048, crand.Reader) // nolint:staticcheck
```
Comment (moul) [inline go/cmd/rdvp/main.go:98]:
```suggestion
				priv, _, err = libp2p_ci.GenerateKeyPairWithReader(libp2p_ci.Ed25519, -1, crand.Reader) // nolint:staticcheck
```

less ambiguous

**Review by moul — APPROVED** [review]

**Pull request merged**